.github/workflows/tag-more-reviewers.yaml

```diff
@@ -0,0 +1,30 @@
+name: Tag more reviewers
+
+on:
+  pull_request:
+    types:
+    - review_requested
+
+jobs:
+  tag-konflux-maintainers:
+    # We have lots of PR traffic from MintMaker (acting as `red-hat-konflux[bot]`), and so it's unsustainable to go
+    # through these emails every day. Therefore, the notifications are disabled for `konflux-maintainers-no-email`
+    # team that's set as owner in CODEOWNERS for the Konflux stuff.
+    # At the same time, we want to be notified when humans, not the bot, request reviews (which happens automatically
+    # again through CODEOWNERS) for the Konflux-related files. This job invites `konflux-maintainers` team for review
+    # for such cases.
+    if: |
+      github.event.requested_team.name == 'konflux-maintainers-no-email' &&
+      github.event.pull_request.user.login != 'red-hat-konflux[bot]'
+    env:
+      GH_TOKEN: ${{ secrets.RHACS_BOT_GITHUB_TOKEN }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Tag Konflux Maintainers for review
+      run: |
+        exec gh api \
```

Comment:
Is `exec` truly needed here? If so, maybe a small comment explaining why that is so might be useful. If not, then we probably want to remove it.

Reply:
Alright then, I wasn't that far off, I just never see `exec` used with GHA, so I was wondering if there was anything special about this particular case. Thanks for the detailed explanation!

Reply:
Note that I got rid of `exec` in the latest revision for consistency with StackRox (https://github.com/stackrox/stackrox/pull/17968).